### PR TITLE
fix: wrong method name, add `cicd:bridge` label

### PR DESCRIPTION
**Pull request by 0xmovses**
# Summary
- `dot_movement.try_get_or_create_config_from_json` has somehow disappeared. 
Quick fix here to keep main building until it is revived.  

# Changelog
- use `try_get_config_from_json`
- Adds bridge label to skip flaky CI

# Testing
CI should pass

### CODE REVIEW

**Review by andygolay — APPROVED** [review]
LGTM

File: networks/suzuka/indexer/src/main.rs [hunk 1]
```diff
@@ -16,8 +16,8 @@ fn main() -> Result<(), anyhow::Error> {
 		.init();
 
 	let dot_movement = dot_movement::DotMovement::try_from_env()?;
-	let maptos_config = dot_movement
-		.try_get_or_create_config_from_json::<maptos_execution_util::config::Config>()?;
+	let maptos_config =
+		dot_movement.try_get_config_from_json::<maptos_execution_util::config::Config>()?;
```
Comment (musitdev) [inline networks/suzuka/indexer/src/main.rs:20]: It's not the right correction. If you do that the config is not created if absent. The methos to call is `try_get_or_create_config_from_json` I don't know why it diseapper.

**Review by musitdev — COMMENTED** [review]
Perhaps I can do a specific PR for the `try_get_or_create_config_from_json` and you merge the CI change?

File: networks/suzuka/indexer/src/main.rs (hunk 1 above)
Comment (0xmovses) [reply networks/suzuka/indexer/src/main.rs:20]: This PR is just to ensure `main` will build, but yes if you could add that follow up PR that would be great.

**Comment by 0xmovses** [comment]
This was just caused my some cached files. After running `cargo clean` the error went away.

**Pull request closed without merging**